src/scripts/main.js

```diff
@@ -50,6 +55,25 @@ function populateSignatures(data, force) {
 	messageBodies
 	.forEach(function (message) {
 		const parent = findParentElementByAttribute(message, 'role', 'dialog');
+		
+		const submitButton = parent.querySelector(".gU.Up");
+		
+		submitButton.addEventListener('click', function(){
+			
+			// Circumstances exist where a user can click send, and the email doesn't send
+			//  - such as forgetting to put a recipient in the dialog. So, when the send
+			// button is clicked, we wait 3 seconds, and see if there's a success dialog i
+			// somewhere in the page.
+			 			
+			setTimeout(function(){
```

Comment:
I'd prefer to trigger an event knowing that our user had clicked something and then use the DOM to check that what we think happened (sending an email) actually happened.